flexget/plugins/parsers/parser_guessit.py

```diff
@@ -177,6 +176,25 @@ def __init__(self, data, name, guess_result, **kwargs):
     def title(self):
         return self._guess_result.get('title')
 
+    def populate_entry_fields(self, entry):
```

Comment:
mutable methods are bit annoying, in this instance you would be better of by returning a dict and using entry.update